src/pages/Dashboard.vue

```diff
@@ -171,6 +184,262 @@ function handleKeyboardShortcuts(e: KeyboardEvent) {
   }
 }
 
+const isMultiple = computed(() => dashboardStore.selectedTorrents.length > 1)
+const hashes = computed(() => dashboardStore.selectedTorrents)
+const hash = computed(() => hashes.value[0])
+const torrent = computed(() => torrentStore.getTorrentByHash(hash.value))
+const torrents = computed(() => dashboardStore.selectedTorrents.map(torrentStore.getTorrentByHash).filter(torrent => !!torrent))
+const availableCategories = computed(() => [{ name: '' }, ...maindataStore.categories])
+
+async function resumeTorrents() {
```

Comment:
Why not put all of this in the separate TRC component? 🤔 

Feels like it's poluting the Dashboard component quite a bit

Reply:
I put it here because I made the TRC component generic to also use it in the content tab without duplicating everything. Not sure where we can put all of that 🤔

Reply:
I thought TRC was 'Torrent Right Click', so that can remain? 🤔 
Which would be the 'implemented version' of your generic right click component?

Reply:
That's why I renamed it RightClickMenu ^^

Reply:
And the TRC menu got removed, so that one can be re-added? But using the generic component you made

Reply:
It's already been extracted. Both the dashboard and content tab use the same RightClickMenu component.

Reply:
Yes, put everything TRC related can be moved back into the TRC component as it existed before? (just with changed syntax)
That way all that code isn't poluting the Dashboard component anymore.

Reply:
Okay I got it, will do that